index.html

```diff
@@ -1,252 +1,252 @@
+            </div>
+
+
+            <div class="rec41">
+                <h3 class="rec41text">Languages</h3>
+                <ul>
+                    <li>JavaScript(ES%/ES6)</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+
+            <div class="rec42">
+                <h3 class="rec41text">Frameworks</h3>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on rails</li>
+                    <li>Rspec</li>
+                    <li>Capibara</li>
+                    <li>Selenium</li>
+                </ul>
+
+            </div>
+            <div class="rec43">
+                <h3 class="rec41text">skills</h3>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Codespan</li>
+                    <li>GitHub</li>
+                    <li>GitLab</li>
+                    <li>Terminal</li>
+                </ul>
+
+            </div>
+
+    </section>
+    <section class="contact">
             <div class="contact-container">
                 <div class="image">
                     <img src="images/image_geometry_1.png" alt="">
                 </div>
                 <div class="contact-heading">
                     <h1 class="heading-contact">Get started</h1>
-                </div>
+                </div>              
                 <div class="contact-para">
                     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
                 </div>
-                <div class="contact-info1">
-                    <p class="p1">your name</p>
-                    <p class="p2">Anastasia</p>
-                </div>
-                <div class="contact-info2">
-                    <p class="p1">your email</p>
-                    <p class="p2">|</p>
-                </div>
-                <div class="contact-info3">
-                    <p class="p2">Enter your message here...</p>
-                </div>
-                <div  class="get-startedcollaboration">
-                <p class="get-started-start-collab">start collaboration <p>
-                </div>
-                <div class="contact-img">
-                    <button class="resume-btn"><a href="#"> Get my Resume  <img class="share-icon" src="images/icons8-share-24.png" alt="union"></a></button>
-                </div>
+                <form action="https://formspree.io/f/xgebyodb" method="post">
+                    <input type="text" class="name" name="name" placeholder="your name" required="true" maxlength="30" > <br>
+                    <input type="email" class="email" name="Email" required="true" placeholder="your email">
+                    <textarea name="text-area" id="your-message" rows="5" class="message" required="true" placeholder="Enter your message..."></textarea>
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, make sure that the message field accepts up to 500 chars ☺️ . This is to ensure the user is limited to writing a wrong message that may burden you to read 📖 .